contracts/database/orm/orm.go

```diff
@@ -11,6 +11,7 @@ type Orm interface {
 	DB() (*sql.DB, error)
 	Query() Query
 	Factory() Factory
+	Model(value any) Orm
```

Comment:
I think we don't need this method, there is a `Model` method in `database/gorm/query.go`, and we should modify it to implement the feature.

Reply:
And some methods can be called directly, for example: 
```
user := models.User{Name: "name"}
facades.Orm().Query().Create(&user)
```
We need to determine whether `user` defines the `Connection` method,

Reply:
> 

Well problem is that we don't have access to connection in query.go. When we call `Query()` method database already initialized.

Reply:
Maybe we can find a way to refresh the connection.